### PR TITLE
Adds cartoon hitsounds for clown outfit

### PR DESCRIPTION
**Pull request by JohnGinnane**
:cl: JohnGinnane
add: Added new sounds for when the clown is hit
tweak: Clown's shoes now squeak 33% faster when moving
/:cl:

Video: https://streamable.com/jfpt3
https://streamable.com/p93wf

Ripped these from DK64, need to add coconut gun at some stage

### CODE REVIEW

**Comment by ghost** [comment]
yes god +1

File: hippiestation/code/modules/clothing/under/jobs/civilian.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/obj/item/clothing/under/rank/clown/Initialize()
+	. = ..()
+	// remove any previous components
```
Comment (ghost) [inline hippiestation/code/modules/clothing/under/jobs/civilian.dm]: i'd argue the comment doesn't reflect on the code here specifically removing squeak components and could lead to confusion in the future

File: hippiestation/code/modules/clothing/under/jobs/civilian.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	var/list/squeaks = GetComponents(/datum/component/squeak)
+	for(var/datum/component/c in squeaks)
+		c.RemoveComponent()
+	AddComponent(/datum/component/squeak, list('hippiestation/sound/misc/hitsounds/cartoon_badoing.ogg'=1,
```
Comment (ghost) [inline hippiestation/code/modules/clothing/under/jobs/civilian.dm]: might being a bit anal here but potentially make a subclass `/datum/component/squeak/clown` or something so we can reuse the same squeak in the future and to take this giant list out of this file?

**Review by ghost — DISMISSED** [review]

File: hippiestation/code/modules/clothing/under/jobs/civilian.dm (hunk 2 above)
Comment (JohnGinnane) [reply hippiestation/code/modules/clothing/under/jobs/civilian.dm]: Good idea

**Comment by Pyko1** [comment]
great, even tho it might be better if it would play the normal hitsound as well. +1

**Comment by ghost** [comment]
+1

**Comment by JohnGinnane** [comment]
Just gonna slap a WIP on this

**Comment by Carbonhell** [comment]
could those sounds cause us any sort of licensing issue

**Comment by JohnGinnane** [comment]
@McBawbaggings I've updated code. I making our own squeak subtypes but the override var is only used when you AddComponent(), so 🤷‍♂️ 

I lowered clowns' shoes squeak delay from 1 to 0 (i.e. 33% reduction)

**Comment by JohnGinnane** [comment]
@Carbonhell Maybe? I don't see how this could cause a licensing issue

**Comment by JohnGinnane** [comment]
This is ready

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
+1 merging

**Pull request merged**